queries/__init__.py

```diff
@@ -25,7 +25,7 @@
 """
 
 from tree import Node
-from typing import Mapping, Optional, List, Dict, Tuple, Union, cast
+from typing import Mapping, Optional, List, Dict, Tuple, Union, cast, Iterable
```

Comment:
Move `from typing import` to the front (before `from tree import`)

Reply:
Fixed!